tests/conftest.py

```diff
@@ -28,6 +30,12 @@ def pytest_addoption(parser):
                           " with loaded modules,"
                           " defaults to `%(default)s`")
 
+    parser.addoption('--cluster-nodes', default=default_cluster_nodes,
```

Comment:
**Nit:** Since all our added options start with redis, might as well do --redis-cluster-nodes to match.